src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/documentation.ts

```diff
@@ -21,6 +21,7 @@ const examplesLabel = i18n.translate('kbn-esql-ast.esql.autocomplete.examplesLab
 export const buildFunctionDocumentation = (
```

Comment:
this file and the following one do not modify our iDoc but the one present in the editor. I don't know if we want to keep it

<img width="1265" height="225" alt="categorize" src="https://github.com/user-attachments/assets/3afa26b8-a42b-4a7a-8a66-cfb056c7008c" />
<img width="1688" height="440" alt="st_extent_agg" src="https://github.com/user-attachments/assets/030fb13a-f3bd-4741-b6fd-9087d6b88f48" />

Reply:
Yes it is nice, lets keep it!